### PR TITLE
do not trigger autocomplete on Enter key

### PR DESCRIPTION
**Pull request by SimonLab**
see #608

At the moment for each search triggered with the "enter" key an unused autocomplete request is also trigger. This PR check the code of the key before executing the autocomplete.

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab I'm _happy_ with this micro-change. 👍

File: client/lib/search-box.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = function () {
   });
   searchinput.addEventListener('keyup', debounce(function (e) {
     var q = e.target.value;
-    if (q.length > 0) {
+    if (q.length > 0 && e.code !== 'Enter') {
```
Comment (SimonLab) [inline client/lib/search-box.js]: I've checked the value linked to ```e``` and it seems that 'Enter' is a right value. I'll check if it's specific to a certain browser and if it exists a better value

Comment (Danwhy) [inline client/lib/search-box.js]: Enter is the [correct value](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values), but maybe we should use [event.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead, as it has better support than [event.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

**Comment by nelsonic** [comment]
Restarted the build: https://travis-ci.org/TheScienceMuseum/collectionsonline/builds/178567744 ... ⌛️

**Comment by nelsonic** [comment]
@SimonLab one test fails on the build: https://travis-ci.org/TheScienceMuseum/collectionsonline/builds/178567744#L4885

**Comment by SimonLab** [comment]
Thanks @nelsonic it seems that the data from Elasticseach as slightly change, I will investigate :+1:

**Comment by nelsonic** [comment]
@SimonLab thanks for confirming that. merging. 👍

**Pull request merged**